### PR TITLE
mZS7aZyr Delete 'Choose technology' page

### PR DESCRIPTION
**Pull request by bravegrape**
The content on this page is either no longer current or covered 
elsewhere.

The 'Use extended validation certificates' section describes a requirement the NCSC no longer imposes.

The 'Use the Verify Service Provider' section has information covered in the [VSP README file](https://github.com/alphagov/verify-service-provider/blob/master/README.md) or in the instructions on how to [get started with the VSP](https://www.docs.verify.service.gov.uk/get-started).

### CODE REVIEW

**Comment by bravegrape** [comment]
[Delete 'Choose Technology' page](https://trello.com/c/mZS7aZyr/181-delete-choose-technology-page)

File: source/plan-your-connection/index.html.md.erb [hunk 1]
```diff
@@ -5,7 +5,7 @@ weight: 20
 
 # Plan your connection
 
-Before starting the development work to integrate GOV.UK Verify into your service, you should [choose the technologies][choose-technology] to help you do this.
+Before starting the development work to integrate GOV.UK Verify into your service, you should choose the technologies to help you do this.
```
Comment (jonathanglassman) [inline source/plan-your-connection/index.html.md.erb]: If this information is covered elsewhere as per the PR description, is it worth linking to that information?

Comment (bravegrape) [reply source/plan-your-connection/index.html.md.erb]: I will re-write this page when I finish re-structuring the planning section (likely next sprint).

There will be no page or section about choosing technologies. That page was created in the past to have a place to put NCSC requirements and to try and get services to use the VSP.

The NCSC requirements have been dropped and with recent changes, new services will be connecting using only the VSP.

Comment (jonathanglassman) [reply source/plan-your-connection/index.html.md.erb]: sounds good

**Pull request merged**